Sources/XCTestHTMLReport/Classes/Models/JoinedRuns.swift

```diff
@@ -16,7 +16,7 @@ struct JoinedRuns : HTML
     let testGroupCollections: [TestGroupCollection]
 
     var numberOfTests : Int {
-        return allTestGroups.count
+        return testGroupCollections.reduce(0, { $0 + $1.testGroups.count })
```

Comment:
What about using the same syntax for reduce like the one used in the `TestGroupCollection.swift`?
`self.duration = testGroups.reduce(0) { $0 + $1.duration }`
Just a suggestion, not needed to change it.